paper.tex

```diff
@@ -35,6 +35,8 @@
 
 \usepackage{longtable}
 
+\hyphenation{Math-e-ma-ti-ca pol-y-morph-ism}
```

Comment:
Yeah, I just saw this at the end of the build log and wanted to see if I could fix it. But let's not worry about it now.